x/erc20/keeper/ibc_callbacks.go

```diff
@@ -138,17 +140,15 @@ func (k Keeper) OnRecvPacket(
 	}
 
 	// For now the only case we are interested in adding telemetry is a successful conversion.
-	defer func() {
-		telemetry.IncrCounterWithLabels(
-			[]string{types.ModuleName, "ibc", "on_recv", "total"},
-			1,
-			[]metrics.Label{
-				telemetry.NewLabel("denom", coin.Denom),
-				telemetry.NewLabel("source_channel", packet.SourceChannel),
-				telemetry.NewLabel("source_port", packet.SourcePort),
-			},
-		)
-	}()
+	telemetry.IncrCounterWithLabels(
```

Comment:
the comment is inaccurate because this also occurs in case that neither of the above switches were accessed